src/cpu/aarch64/acl_binary.hpp

```diff
@@ -123,6 +123,10 @@ struct acl_binary_t : public primitive_t {
 
         DECLARE_COMMON_PD_T("acl", acl_binary_t);
 
+        pd_t(const binary_desc_t *adesc, const primitive_attr_t *attr,
+                const binary_pd_t *hint_fwd_pd)
+            : cpu_binary_pd_t(adesc, attr, hint_fwd_pd) {}
+
```

Comment:
This was put here so that `acl_post_ops_t` could construct the pd, as the constructor is protected. I have made it a friend struct instead as this seems more consistent. Similarly for eltwise.